test/over_react/util/react_wrappers_test.dart

```diff
@@ -227,16 +227,23 @@ main() {
       });
 
       group('updates the "key" and "ref" props properly', () {
-        const Map originalKeyRefProps = const {
+        var originalRefCalled = false;
+        var cloneRefCalled = false;
+        Map originalKeyRefProps = {
           'key': 'original',
-          'ref': 'original'
+          'ref': (ref) { originalRefCalled = true; }
```

Comment:
These were changed to callback refs because string refs are considered legacy and will be deprecated eventually